### ISSUE TITLE
[BUG] Wrong hitbox on Arch rounded (corner), wedge/corner/hepta rounded and window arch rounded (corner) bricks

### ISSUE THREAD

**Issue by Uncle-Ulty** [tsunamayo/Starship-EVO#5456]
Lot's of recent added bricks got their hitbox in strange shapes, which doesn't allow us to properly use them.

The issue happens in the following bricks
-arch rounded
-arch rounded corner

-wedge rounded
-corner rounded
-hepta rounded

-window arch rounded
-window arch rounded corner

![image](https://github.com/tsunamayo/Starship-EVO/assets/73253472/7071a595-a97e-4fe2-a38f-e157f347bf36)

In all the aforementioned bricks, the hitbox doesn't follow the concave surface of the brick. Instead, it follows a straight line, which forbids us to place any brick next to the rounded brick.

![image](https://github.com/tsunamayo/Starship-EVO/assets/73253472/5fa6865c-57ef-42c2-ae03-79e741f752cc)
_(I can't place the brick in the left image, but I can place the brick on the right image)_

There is a "dead zone" between the inner concave surface and this straight line.

The same happens with the wedge and glass.

![image](https://github.com/tsunamayo/Starship-EVO/assets/73253472/f2ec695b-aaf2-4f01-8fab-340410c1e416)

![image](https://github.com/tsunamayo/Starship-EVO/assets/73253472/cf4f452e-569e-4b9c-9890-c466a513a25c)

This issue is more severe on corner bricks such as window arch rounded corner:

![image](https://github.com/tsunamayo/Starship-EVO/assets/73253472/549a835c-417b-45e6-bc0e-61b4e7b7b3d7)

These hitbox's deadzones make these bricks less versatile to use in tighter spaces, such as cockpit and ball turrets

I hope it's something easy to fix

:D

**Comment by TIKIRobo** [comment]
#5322 yeah this bug has been around for a bit, hopefully Tsuna is able to find a fix soon!

**Comment by CoasterBlock** [comment]
Yeah this is a very persistent issue. Its nothing new,

**Comment by Uncle-Ulty** [comment]
Yep, it's an old issue, but since it still exists, I thought it wasn't reported before. Thanks for warning me, cheers
